maxim/logger/mistral/client.py

```
@@ -0,0 +1,215 @@
+                    for chunk in c.get("choices", [])
+                )
+                trace.set_output(text)
+                trace.end()
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][MaximMistralChat] Error in logging generation: {e}"
+            )
+
+
+class MaximMistralClient:
+    def __init__(self, client: Mistral, logger: Logger):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)